### PR TITLE
[SQL] Add diamond_buckler and mods for kupo_suit

### PR DESCRIPTION
**Pull request by CatsEyeXI**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Add diamond_buckler and mods for kupo_suit

## Steps to test these changes

!additem 25726
equip

### CODE REVIEW

File: sql/item_mods.sql [hunk 1]
```diff
@@ -53905,7 +53905,11 @@ INSERT INTO `item_mods` VALUES (25718,165,3);   -- CRITHITRATE: 3
 INSERT INTO `item_mods` VALUES (25718,384,400); -- HASTE_GEAR: 400
 
 -- Jubilee Shirt
-INSERT INTO `item_mods` VALUES (25722,1,1); -- DEF: 1
+INSERT INTO `item_mods` VALUES (25722,1,1);     -- DEF: 1
+
+-- Kupo Suit
+INSERT INTO `item_mods` VALUES (25726,169,18);  -- Movement speed: +18%
```
Comment (claywar) [inline sql/item_mods.sql:53911]: Please use the exact enum values in comments

Comment (CatsEyeXI) [reply sql/item_mods.sql:53911]: Would that be MOVEMENT_SPEED? I'm coming up empty-handed searching through the repo for examples...

image

Comment (claywar) [reply sql/item_mods.sql:53911]: You're referencing item_latents in regards to item_mods.sql.  Please check the updated format, and it _must_ match the enum name and value for what you're adding.

Comment (CatsEyeXI) [reply sql/item_mods.sql:53911]: Nevermind, think I found it...

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**